src/main/groovy/com/gorylenko/PropertiesFileWriter.groovy

```diff
@@ -25,6 +22,27 @@ class PropertiesFileWriter {
             Collections.sort(v)
             return new Vector<Object>(v).elements()
         }
+
+        @Override
+        void store(OutputStream out, String comments) throws IOException {
+            store(new BufferedWriter(new OutputStreamWriter(out, "8859_1")), comments)
+        }
+
+        @Override
+        void store(Writer writer, String comments) throws IOException {
```

Comment:
Thanks a lot for the PR. It would be better if we could improve it a little by implementing something like StripFirstLineStream (see https://stackoverflow.com/questions/6184335/properties-store-suppress-timestamp-comment/6184414#6184414) because it would help to avoid issues regarding platform encoding, memory buffer usage and support platform-dependent line ending?

Reply:
Hi,

sorry for late reply, i've been busy; I've just pushed commit 2bd7022, which adds platform-dependent newline support to generated properties file.

I've been thinking about using special stream/writer implementation that would prevent writing comments to actual output stream and i concluded that these solutions are not worth the effort in _my opinion_. My arguments are simple: git properties task is ran only once per gradle build and it handles very small amount of data (in my project generated `git.properties` file is only 151 bytes).